src/utils.spec.js

```diff
@@ -189,9 +195,20 @@ describe('getCSS', () => {
       borderBottomWidth: 2 * 4,
     })
   })
+
+  it('should accept a key in spacingNames and return the pixel value', () => {
+    const css = getCSS('marginBottom', 'XL')
+
+    expect(css).toEqual({
+      borderBottomWidth: spacingNames.XL,
+    })
+  })
 })
 
 describe('parseSpacing', () => {
+  it('should return undefined if spacing is undefined', () => {
+    expect(parseSpacing(undefined)).not.toBeDefined()
```

Comment:
This case only happens in test as far as I can tell and adds additional logic to the code, let's remove it

Reply:
This is actually valid.  This is the correct default behavior when a Grid/Col component is called with no margin or padding properties.  I'm going to leave this test here.

Reply:
You are right, this test makes sense